users/views.py

```diff
@@ -1,3 +1,11 @@
-# from django.shortcuts import render
+from rest_framework import generics
+from rest_framework.permissions import AllowAny
 
-# Create your views here.
+from users.models import User
+from users.serializers import SignupSerializer
+
+
+class SignupView(generics.CreateAPIView):
+    queryset = User.objects.all()
```

Comment:
Looks like this line is excess. This view does not use it as it does not have retrieve or list endpoints

Reply:
Removed